pykafka/producer.py

```diff
@@ -374,13 +374,22 @@ def _produce(self, message):
         :type message: `pykafka.protocol.Message`
         """
         success = False
+        retry = 0
+        max_retries = self._max_retries * 5
```

Comment:
What is the value of self._max_retries at this point?

Reply:
Its 3. https://github.com/Reflektion/pykafka/blob/3315d93081c7e9f2b6f857d97052fe24bc36f3ce/pykafka/producer.py#L63